ansible_deployer/command_line.py

```diff
@@ -200,15 +200,51 @@ def check_cfg_permissions_and_owner(cfg_path: str):
         logger.error("Program will exit now.")
         sys.exit(41)
 
+def get_config_paths():
+    """Function to create absolute config paths and check their extension compatibility"""
+    ymls = []
+    yamls = []
+    infra_cfg = None
+    tasks_cfg = None
+
+    for config in os.listdir(conf["global_paths"]["config_dir"]):
+        if config != "ansible-deploy.yaml":
+            if config.endswith(".yml"):
+                ymls.append(config)
+            elif config.endswith(".yaml"):
+                yamls.append(config)
+
+            if config.startswith("infra"):
+                infra_cfg = os.path.join(conf["global_paths"]["config_dir"], config)
+            elif config.startswith("tasks"):
+                tasks_cfg = os.path.join(conf["global_paths"]["config_dir"], config)
+
+    if len(ymls) > 0 and len(yamls) > 0:
+        logger.critical("Config extensions are not compatible!")
+        logger.debug("Config files with yml extensions: %s", " ".join(ymls))
+        logger.debug("Config files with yaml extensions: %s", " ".join(yamls))
+        sys.exit(42)
+
+    if not infra_cfg:
+        logger.critical("Infrastructure configuration does not exist!")
```

Comment:
maybe something like:
("Failed to open infrastructure configuration file in "%s" does not exist", conf["global_paths"]["config_dir"]) ?